frontend/model/contracts/group.js

```diff
@@ -401,8 +401,7 @@ DefineContract({
         paymentHash: string,
         updatedProperties: objectMaybeOf({
           status: paymentStatusType,
-          details: object,
-          memo: string
```

Comment:
Why is it needed? I didn't see any place using it. The UI only needs `memo` at `group/payment`.

Reply:
@sandrina-p Everything in this file involves thought that has been extended out like 5 years into the future. Just because something isn't being used right now doesn't mean it won't be used in the future. :P

Different payment systems support different levels of information, and if we want to be able to support all payment systems plus our own metadata and notes, we need this field. Please add it back.

Reply:
I did not know about that, it wasn't explicit anywhere. I'll add it back then.